src/main/java/com/powsybl/caseserver/elasticsearch/ESConfig.java

```diff
@@ -69,23 +68,22 @@ public ElasticsearchCustomConversions elasticsearchCustomConversions() {
     }
 
     @WritingConverter
-    enum DateToStringConverter implements Converter<DateTime, String> {
+    enum DateToStringConverter implements Converter<ZonedDateTime, String> {
         INSTANCE;
 
         @Override
-        public String convert(DateTime date) {
-            return date.toDateTimeISO().toString();
+        public String convert(ZonedDateTime date) {
+            return date.format(DateTimeFormatter.ISO_OFFSET_DATE_TIME);
```

Comment:
ISO_OFFSET_DATE_TIME because it's a ZonedDateTime ?
Why not ISO_DATE_TIME ? ISO_ZONED_DATE_TIME ?

Reply:
This is to match the previous implementation (i.e toDateTimeISO()). 
![image](https://github.com/powsybl/powsybl-case-server/assets/123802855/3d683ab2-d300-486a-ada0-9f47a2b24636)
We do not want the timezone between brackets, that's why we use ISO_OFFSET_DATE_TIME